index.js

```diff
@@ -306,4 +306,22 @@ Notifications.clearAllNotifications = function() {
 	return this.callNative('clearAllNotifications', arguments)
 }
 
+Notifications.removeAllDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('removeAllDeliveredNotifications')
+  }
+}
+
+Notifications.removeDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('removeDeliveredNotifications', arguments)
+  }
+}
+
+Notifications.getDeliveredNotifications = function() {
+  if ( Platform.OS === 'ios' ) {
+    return this.callNative('getDeliveredNotifications', arguments)
+  }
+}
+
 module.exports = Notifications;
```

Comment:
@gperdomor Great stuff 👍  and thanks for the clarification. I missed your earlier comment in PR which created the confusion. Sorry about that. Looking forward to the react-native PR getting merged. I checked the react-native PR. Out of curiosity, how do I get the identifier for a notification to feed to the `removeDeliveredNotifications` that you added to `PushNotificationIOS` package.